mitxgraders/plugins/integralgrader.py

```diff
@@ -218,23 +220,18 @@ def schema_config(self):
             Required('user_constants', default={}): validate_user_constants(Number),
             # Blacklist/Whitelist have additional validation that can't happen here, because
             # their validation is correlated with each other
-            Required('blacklist', default=[]): [str],
+            Required('blacklist', default=[]): [text_string],
             Required('whitelist', default=[]): Any(
                 All([None], Length(min=1, max=1)),
-                [str]
+                [text_string]
             ),
             Required('tolerance', default='0.01%'): Any(PercentageString, NonNegative(Number)),
             Required('samples', default=1): Positive(int),  # default changed to 1
-            Required('variables', default=[]): All([str], all_unique),
+            Required('variables', default=[]): All([text_string], all_unique),
             Required('sample_from', default={}): dict,
             Required('failable_evals', default=0): NonNegative(int)
         })
 
-    schema_user_consts = All(
```

Comment:
Were we just not using this?

Reply:
That's correct, is was unused... the schema above used the imported `schema_user_functions`.